### ISSUE TITLE
Replace RSS icon with an Icon to be used for login

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1117]
Priority 14 (0	3	5	6)
The size is an estimate based on simple changes
Idea for a solution to the 'help' tab being lost on smaller screens: remove the RSS icon and use an icon to replace the 'member' tab.  Possibly use a keyhole for the icon.  New volunteers (and new registered researches) would be informed that they should use the icon to reach the login page.

**Comment by PatReynolds** [comment]
But buttons are not visible on a mobile phone.

**Comment by richpomfret** [comment]
Let's mock up a design for this to review. To consider different screen sizes (would the icons still appear on mobile?).

**Comment by DeniseColbert** [comment]
Poosibly remove RSS feed button, relocate Github button to footer and keep Member link to log in

**Comment by PatReynolds** [comment]
Due to changes in the tabs, I think this is no longer needed - moving to Needs review
